tests/test_anm.py

```diff
@@ -190,30 +189,33 @@ def test_frequency_fluctuation_dcc(ff_name):
             # no temperature weighting
             tem_scaling = 1
             tem = 1
+            reference_fluc = np.genfromtxt(join(data_dir(), ref_fluc),
+                                            skip_header=1, delimiter=","
+                                        )
```

Comment:
In other places in the code base, and as far as I have seen, most commonly in Python projects, the indentation of a line with a closing bracket bracket is the same as the corresponding opening bracket.

```suggestion
            reference_fluc = np.genfromtxt(
                join(data_dir(), ref_fluc),
                skip_header=1, delimiter=","
            )
```